waspc/data/Generator/templates/react-app/src/queries/core.d.ts

```diff
@@ -1,3 +1,7 @@
-export type Query<Input, Output> = (args: Input) => Promise<Output>
+export type Query<Input, Output> = {
+  (args: Input): Promise<Output>
+  queryCacheKey: string[]
+  route: { method: string, path: string}
```

Comment:
I should probably make `method` a proper type... does anyone know of anything in Node/Express/Axios I can make use of for a `'GET'` etc. constant equivalent?

Reply:
I would suggest defining a type like `type Method = "GET" | "POST" | ...`